tests/storage/export.go

```diff
@@ -390,6 +396,7 @@ var _ = SIGDescribe("Export", func() {
 		out, stderr, err := exec.ExecuteCommandOnPodWithResults(virtClient, downloadPod, downloadPod.Spec.Containers[0].Name, md5Command(fileAndPathName))
 		Expect(err).ToNot(HaveOccurred(), out, stderr)
 		md5sum := strings.Split(out, " ")[0]
+		Expect(md5sum).To(HaveLen(32))
```

Comment:
why do we need this check? we check in the next line that it is equal to the expected md5

Reply:
imagine if they are both empty for some reason
just a precaution